internal/endtoend/testdata/mysql_enums/go/models.go

```diff
@@ -0,0 +1,47 @@
+// Code generated by sqlc. DO NOT EDIT.
+
+package querytest
+
+import ()
+
+type FirstNameType string
+
+const (
+	john   FirstNameType = "john"
+	albert FirstNameType = "albert"
```

Comment:
We're going to have to fix the naming here. These should be

```go
const (
	FirstNameJohn   FirstNameType = "john"
	FirstNameAlbert FirstNameType = "albert"
)
```

Reply:
It's fine for now, will address it in another PR.

Reply:
Good find. I must have written some unfortunately narrow tests that didn't catch that before.